modules/library/render/src/test/resources/org/geotools/renderer/lite/test-data/margin/mark-margin.sld

```diff
@@ -20,7 +20,7 @@
                 </Graphic>
               </GraphicFill>
             </Fill>
-            <VendorOption name="graphic-margin">5</VendorOption>
+            <VendorOption name="graphic-margin">20</VendorOption>
```

Comment:
Can we have a new test, rather than reusing the existing one? 
So that the test covers the case of margin smaller than symbol, and the case of marging bigger than symbol, along with ensure that existing styles keep on working in the same way?